packages/e2e-tests/specs/editor/plugins/image-size.test.js

```diff
@@ -57,19 +60,25 @@ describe( 'changing image size', () => {
 		// Select the new size updated with the plugin.
 		await openDocumentSettingsSidebar();
 
-		const [ imageSizeLabel ] = await page.$x(
+		const imageSizeLabel = await page.waitForXPath(
 			'//label[text()="Image size"]'
 		);
 		await imageSizeLabel.click();
 		const imageSizeSelect = await page.evaluateHandle(
 			() => document.activeElement
 		);
-		imageSizeSelect.select( 'custom-size-one' );
+		await imageSizeSelect.select( 'custom-size-one' );
+
+		await page.waitForSelector( '.wp-block-image.size-custom-size-one' );
 
-		const imageWidth = await page.$eval(
+		// Disable reason: Wait for the input element to update with the new
+		// value, otherwise it can still contain the old one.
+		// eslint-disable-next-line no-restricted-syntax
+		await page.waitFor( 100 );
```

Comment:
Isn't there another way to do this? We could try requestAnimationFrame or requestIdleCallback? At worst, we could use waitForFunction to check if it's 499.

Reply:
It looks like there is 😃 I will update tomorrow. 

I copied it from other tests 🤷‍♂️

Reply:
Sounds good. There's just no reason to delay execution for 100 ms if it's updated quicker :)